### DISCUSSION TITLE
vaultwarden only accessible from the internal network,but sends also from the Internet

### DISCUSSION THREAD

**Discussion by SchwarzerA** [dani-garcia/vaultwarden#4488] [Q&A]
With these settings my reverse proxy works flawlessly but I need to open it to the internet for sending and using sends. Dr. Google gave me no hint so I would like to ask you for assistance with that **Location**-line. What else needs to be added to make sends work from outside (internet)?

---[ 8< ]---
```
…
 <Location ~ "^/(?!.well-known|#/send)">
  Require			ip  <ip>,...
 </Location>
…
 RewriteEngine			on
 RewriteCond			%{HTTP:Upgrade}		=websocket			[NC]
 RewriteRule			/notifications/hub(.*)	ws://<ip>:51081/$1		[P,L]

 ProxyPass			/.well-known			!
 ProxyPass			/				http://<ip>:51080/
 ProxyPreserveHost		on
 ProxyRequests			off
 RequestHeader			set	X-Real-IP		%{REMOTE_ADDR}s
 RequestHeader			add	X-Forwarded-Proto	https
…

```---[ 8< ]---

THank you in advance, any help is appreciated.

**Comment by stefan0xC** [comment]
That won't work because the URI fragments are not send to the server by the browser in the first place. If you want to know what's required for allowing sends to work you should open a Send link with the network console open (with cache disabled) and check what is loaded. Cf. https://github.com/dani-garcia/vaultwarden/discussions/4361
